app/genesis.go

```diff
@@ -69,6 +74,35 @@ func NewDefaultGenesisState() GenesisState {
 	slashingGenesis.Params.DowntimeJailDuration = 60 * 10 * time.Second       // 10 minutes
 	slashingGenesis.Params.SlashFractionDoubleSign = sdk.NewDecWithPrec(5, 2) // 5%
 	slashingGenesis.Params.SlashFractionDowntime = sdk.NewDecWithPrec(1, 4)   // 0.01%
+
+	icaGenesis.HostGenesisState.Params = icahosttypes.Params{
+		HostEnabled: true,
+		AllowMessages: []string{
+			sdk.MsgTypeURL(&authz.MsgExec{}),
```

Comment:
Note: Should we allow our message module too?

Reply:
@colmazia Please note to add our oracle meesages to upgradeHandler 2.5

Reply:
It will be added in the upgradeHandler 2.5 PR.